client/src/utils/components/dateFilter/DateFilter.tsx

```diff
@@ -0,0 +1,144 @@
+  const [dates, setDates] = useState<DatesFilter>({});
+
+  useEffect(() => {
+    if (handler && selectedType == dateFilterTypes.custom)
+      handler(dates);
+  }, [dates]); // eslint-disable-line
+
+  const changeDateType = (typeDate: dateFilterTypes) => {
+    let from, to;
+    setSelectedType(typeDate);
+    const today = new Date();
+    switch (typeDate) {
+      case dateFilterTypes.last90days:
+        to = today;
+        from = new Date(new Date().setDate(to.getDate() - 90));
+        break;
+      case dateFilterTypes.lastMonth:
+        to = today;
+        from = new Date(new Date().setDate(to.getDate() - 30));
+        break;
+      case dateFilterTypes.lastWeek:
+        to = today;
+        from = new Date(new Date().setDate(to.getDate() - 7));
+        break;
+      case dateFilterTypes.oldest:
+        to = new Date(new Date().setDate(today.getDate() - 90));
+        break;
+      default:
+        from = "";
+        to = "";
+        break;
+    }
+    if (handler && typeDate != dateFilterTypes.custom) {
+      handler({
+        since: from ? Time.toIsoString(from, "date") : "",
+        until: to ? Time.toIsoString(to, "date") : "",
+      });
+    }
+  };
+
+  const items = [
+    { title: "All", value: dateFilterTypes.all },
+    { title: "Last Week", value: dateFilterTypes.lastWeek },
+    { title: "Last Month", value: dateFilterTypes.lastMonth },
+    { title: "Last 90 days", value: dateFilterTypes.last90days },
+    { title: "Oldest", value: dateFilterTypes.oldest },
+    { title: "Custom", value: dateFilterTypes.custom },
+  ];
+  const datesInput = selectedType === dateFilterTypes.custom ?
+    <>
+      <div>
+        <label className="px-2 author-label">From:</label>
+        <Input type="date" name="start"
+          max={new Date().toISOString().split("T")[0]}
+          onChange={(e: ChangeEvent<HTMLInputElement>) => setDates({ ...dates, since: e.target.value })}/>
+      </div>
+      <div>
+        <label className="px-2 author-label">To:</label>
+        <Input type="date" name="end"
+          max={new Date().toISOString().split("T")[0]}
+          onChange={(e: ChangeEvent<HTMLInputElement>) => setDates({ ...dates, until: e.target.value })}/>
+      </div>
+    </>
+    : null;
+
+  const options = items.map(item => {
+    const nameInput = `author-${item.value}`;
+    return (
+      <div className="d-flex align-items-center" key={nameInput}>
+        <Input
+          type="radio"
+          name="date-filter"
+          value={item.value}
+          onChange={(e: ChangeEvent<HTMLInputElement>) => changeDateType(e.target.value as dateFilterTypes)}
+          className="author-input"
+          checked={selectedType === item.value}
+          data-cy={nameInput}/>
+        <label className="px-2 author-label">{item.title}</label>
+      </div>
+    );
+  });
+  return (
+    <>
+      <h3 className="filter-label">Date</h3>
```

Comment:
I think we should be more explicit about what we are filtering on.

```suggestion
      <h3 className="filter-label">Creation Date</h3>
```
```suggestion
      <h3 className="filter-label">Date</h3>
```